pkgs/tools/virtualization/awsebcli/default.nix

```diff
@@ -81,7 +59,7 @@ in with localPython.pkgs; buildPythonApplication rec {
   doCheck = false;
 
   propagatedBuildInputs = [
-    blessed botocore cement colorama docker dockerpty docopt pathspec pyyaml
+    blessed botocore cement colorama dockerpty docopt pathspec pyyaml
```

Comment:
Maybe you could add a comment
```
# FIXME: Add optional docker dependency, which requires requests >= 2.14.2.
# Otherwise, awsebcli will try to install it using pip.
```